pauthabielf64/pauthabielf64.rst

```diff
@@ -231,6 +231,9 @@ changes to the content of the document for that release.
   | 2021q3     | 1\ :sup:`st` November 2021  | Clarify the concepts of a default and alternate PAuth elf signing|
   |            |                             | schemes.                                                         |
   +------------+-----------------------------+------------------------------------------------------------------+
+  | 2023q3     | 16\ :sup:`st` August 2023   | Update DT_AARCH64_AUTH_ values to avoid conflict with            |
```

Comment:
The DT_AARCH64_AUTH_ (I think it is the trailing _ in DT_AARCH64_AUTH_ ) is causing the word to be interpreted as a link which doesn't exist. This is then failing the rstcheck.

I suggest: Update tags in `Dynamic Sections`_ to avoid conflict with DT_AARCH64_VARIANT_PCS.